src/logging.cc

```diff
@@ -154,6 +154,10 @@ GLOG_DEFINE_int32(logbuflevel, 0,
                   " ...)");
 GLOG_DEFINE_int32(logbufsecs, 30,
                   "Buffer log messages for at most this many seconds");
+
+GLOG_DEFINE_int32(logcleansecs, 300,
+                  "Clean overdue logs every this many seconds");
```

Comment:
What's the reason for choosing a signed integer to represent the time span? Isn't it always positive?

Reply:
Yes, it's always positive.
I use integer because another timer ( e.g. `next_flush_time_` ) use integer, too.

```cpp=
void LogFileObject::FlushUnlocked(){
  if (file_ != NULL) {
    fflush(file_);
    bytes_since_flush_ = 0;
  }
  // Figure out when we are due for another flush.
  const int64 next = (FLAGS_logbufsecs
                      * static_cast<int64>(1000000));  // in usec
  next_flush_time_ = CycleClock_Now() + UsecToCycles(next);
}
```